Automattic/woocommerce-payments
# Add unit tests for UPE checkout utility scripts

#### Changes proposed in this Pull Request

This PR adds a new set of JavaScript unit tests to cover the functionality of the UPE checkout utility scripts found [here](https://github.com/Automattic/woocommerce-payments/blob/develop/client/checkout/utils/upe.js). 

### Testing Instructions

Run `npm run test:js -- client/checkout/utils/test/upe.test.js` and check that all tests are passing.

-------------------


**Post merge**
